packages/client/src/features/tools/change-bounds/change-bounds-tool.ts

```diff
@@ -326,23 +310,9 @@ export class ChangeBoundsListener extends DragAwareMouseListener implements ISel
         }
     }
 
-    dispose(): void {
-        this.reset();
-    }
-
     protected reset(): void {
         if (this.activeResizeElement && isResizable(this.activeResizeElement)) {
-            if (this.initialBounds) {
-                const resetResizeAction = SetBoundsAction.create([
-                    {
-                        elementId: this.activeResizeElement.id,
-                        newPosition: this.initialBounds,
-                        newSize: this.initialBounds
-                    }
-                ]);
-                this.tool.deregisterFeedback([resetResizeAction], this);
-            }
-            this.tool.deregisterFeedback([HideChangeBoundsToolResizeFeedbackAction.create()], this);
+            this.tool.registerFeedback([HideChangeBoundsToolResizeFeedbackAction.create()], this);
```

Comment:
Shouldn't this be a `deregisterFeedback`?

Reply:
Yes!